src/main/java/org/ow2/proactive/connector/iaas/rest/InstanceRest.java

```diff
@@ -72,12 +76,17 @@ public Response getInstances(@PathParam("infrastructureId") String infrastructur
             @QueryParam("allInstances") Boolean allInstances) {
 
         if (Optional.ofNullable(instanceId).isPresent()) {
+            log.debug("Receive get request for infrastructure " + infrastructureId + " and instance id " + instanceId);
             return Response.ok(instanceService.getInstanceById(infrastructureId, instanceId)).build();
         } else if (Optional.ofNullable(instanceTag).isPresent()) {
+            log.debug("Receive get request for infrastructure " + infrastructureId + " and instance tag " +
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal " and instance tag " 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)